index.html

```diff
@@ -1504,57 +1504,69 @@
 
 Privacy principles are often defined in terms of extending rights to individuals. However, there are
 cases in which deciding which principles apply is best done collectively, on behalf of a group.
-
-One such case, which has become increasingly common with widespread profiling, is that of information
-relating to membership of a group or to a group's behaviour, as detailed in
-<a href="#group-privacy"></a>. As Brent Mittelstadt explains, “<em>Algorithmically grouped
-individuals have a collective interest in the creation of information about the group, and actions
-taken on its behalf.</em>” ([[?Individual-Group-Privacy]]) This justifies ensuring that grouped
-people can benefit from both individual and collective means to support their [=autonomy=] with respect
-to [=data=] [=processing=]. It should be noted that [=processing=] can be unjust even if individuals
-remain anonymous, not from the violation of individual [=autonomy=] but because it violates ideals
-of social equality ([[?Relational-Governance]]).
-
-Another case in which collective decision-making is preferable is for [=processing=] for which
-informed individual decision-making is unrealistic (due to the complexity of the processing, the
-volume or frequency of processing, or both). Expecting laypeople (or even experts) to make informed
-decisions relating to complex [=data=] [=processing=] or to make decisions on a very frequent
-basis — even if the [=processing=] is relatively simple — is unrealistic if we also want them to have
-reasonable levels of [=autonomy=] in making these decisions.
-
-The purpose of this principle is to require that [=data governance=] provide ways to distinguish
-[=appropriate=] [=data=] [=processing=] without relying on individual decisions whenever the latter
-are impossible, which is often ([[?Relational-Governance]], [[?Relational-Turn]]).
-
-Which forms of collective governance are recognised as legitimate will depend on domains. These may
-take many forms, such as governmental bodies at various administrative levels, standards
+Collective decision-making is likely preferable:
```

Comment:
We could soften the whole recommendation using:

```suggestion
Collective decision-making should be considered:
```